themes/gfsc/layouts/index.html.html

```diff
@@ -56,8 +56,14 @@ <h3 class="window__header">{{ .Title }}</h3>
               <p class="window__content">{{ .Description }}</p>
               <div class="window__footer">
                 <div class="window__footer__buttonbox">
-                  <a class="window__footer__button btn" href="{{ .Permalink }}"
-                    >Browse</a
+                  <a
+                    class="window__footer__button btn"
+                    href="{{ .Permalink }}"
+                    aria-label="Browse {{ .Title }} projects"
+                    >Browse
+                    <span class="screen-reader-only"
```

Comment:
I think you need to pick one way of doing this or the other - I think the screenreader will ignore the screenreader only class here in favour of the aria-label. `Browse about {{.Title}}` doesn't quite make sense to me?

Reply:
I have removed all the hidden text / aria labels